static/usr/lib/systemd/system-generators/serial-console-conf

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -eux
+
+NORMAL_DIR="${1}"
+EARLY_DIR="${2}"
+LATE_DIR="${3}"
+
+for tty in $(cat /sys/class/tty/console/active); do
+    case "${tty}" in
+      tty[0-9]*)
+      ;;
+      *)
+        mkdir -p "${NORMAL_DIR}/getty.target.wants"
+        ln -sf /usr/lib/systemd/system/serial-console-conf@.service "${NORMAL_DIR}/getty.target.wants/serial-console-conf@${tty}.service"
```

Comment:
I understand the commit message on why we do not do the same for vts, but still not sure - my worry is that maybe in the future we could have something different to console=tty1 in UC's kernel command line. Especially as we actually have tty0 usually on classic.